libraries/ms-drivers/test/test_pca9539r_gpio_expander.c

```diff
@@ -308,3 +310,17 @@ void test_pca9539r_gpio_get_state_invalid_address(void) {
   Pca9539rGpioState state;
   TEST_ASSERT_EQUAL(STATUS_CODE_INVALID_ARGS, pca9539r_gpio_get_state(&address, &state));
 }
+void prv_test(const struct GpioAddress *address, void *context) {
```

Comment:
Nitpick: the convention in this file (not everywhere, just in this file) is to separate tests for each function with a comment like `// pca9539r_gpio_get_state` at the start of the `pca9539r_gpio_get_state` tests; please conform to this.